src/models/m_rsc.erl

```diff
@@ -761,7 +761,7 @@ is_a(Id, Context) ->
     m_category:is_a(RscCatId, Context).
 
 %% @doc Return the categories and the inherited categories of the resource. Returns a list with category ids
```

Comment:
Line 763 is too long: %% @doc Return the categories and the inherited categories of the resource. Returns a list with category ids.